tenable/io/v3/was/vulnerability/api.py

```diff
@@ -0,0 +1,140 @@
+'''
+Vulnerability
+=============
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`vulnerabilities <was-v2-vulnerabilities>` API.
+
+Methods available on ``tio.v3.was.vulnerabilities``:
+
+.. rst-class:: hide-signature
+.. autoclass:: VulnerabilityAPI
+    :members:
+'''
+from typing import Dict, Union
+
+from requests import Response
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.base.iterators.explore_iterator import (CSVChunkIterator,
+                                                           SearchIterator)
+
+
+class VulnerabilityAPI(ExploreBaseEndpoint):
+    '''
+    API class containing all the methods related to WAS Vulnerability.
+    '''
+    _path = 'api/v3/findings/vulnerabilities/webapp'
+    _conv_json = True
+
+    def get_details(self, id: str) -> Dict:
+        '''
+        Return details of a particular finding
+
+        Args:
+            id: The unique identifier of the finding.
+
+        Returns:
+            :obj:`dict`: details of the respective finding id
+
+        Examples:
+            >>> tio.v3.was.vulnerabilities.get_details(id)
+        '''
+        return self._get(f'{id}')
```

Comment:
Let's use _details from ExploreBaseEndpoint if the functionalities are same and CSV export should also be supported for this.